spec/requests/api/v0/get_manifest_spec.rb

```diff
@@ -14,6 +14,17 @@
         expect(response).to match_response_schema("get-manifest")
       end
 
+      it 'finds by manifest tracking number' do
+        manifest = create(:manifest, activity_id: 1, document_id: 2)
+        manifest.reload
+
+        get "/api/v0/manifests/#{manifest.tracking_number}",
```

Comment:
do you think it makes sense to remove the ability to GET a manifest by UUID? Not sure why someone would prefer to find via UUID rather than tracking number